players/base.py

```diff
@@ -383,7 +378,8 @@ def escaped(self) -> bool:
     def is_defending(self) -> bool:
         defend_byte = self._read_char_battle_state_address(offset=PlayerMagicNumbers.DEFENDING)
         result = (defend_byte >> 3) & 1 == 1
-        logger.warning(f"Character is defending: {result}")
+        if self.id != memory.main.get_battle_char_turn():
+            return 9
```

Comment:
I feel like typing should be unhappy with this. We are telling this method that it's going to return a `bool` and this is returning an `int`.

Reply:
Typing fixed. @jlward does it look better now?